src/main/java/de/hysky/skyblocker/utils/ItemUtils.java

```diff
@@ -218,6 +218,19 @@ public static Matcher getLoreLineIfMatch(ItemStack item, Pattern pattern) {
         return null;
     }
 
+    @Nullable
+    public static Matcher getLoreLineIfContainsMatch(ItemStack item, Pattern pattern) {
```

Comment:
It would be nice to javadoc this, e.g. it uses `Matcher#find`. You can also javadoc the other methods too.

Reply:
Done in [`478451f`](https://github.com/SkyblockerMod/Skyblocker/pull/728/commits/478451fdc0a24b1b3616ca1d43c19fc5b90cfe70).